frontend/revive-gallery-app/src/components/loginform.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import { Link, useNavigate   } from "react-router-dom";
-import "./loginform.css";
+import "./LoginForm.css";
```

Comment:
@anjaliputta It seems you attempted to change the file names from **loginform.css** to **LoginForm.css**, but git is case-insensitive; it considers both files as the same. So this file renaming will cause many errors for others. 
So I suggest you update the import statements with their previous states.

Reply:
@SaiKiran0407 Yes, thank you for noticing it. I got it now and will update the import statements wherever necessary.